lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -760,19 +801,34 @@ def _bulk_dispatch(
             ApiMethod.POST, ParamType.DATASET, params
         ).value
 
-        operation = (
-            Database.db_session.query(Operation)
-            .filter(
-                Operation.dataset_ref == dataset.id,
-                Operation.state == OperationState.WORKING,
-            )
-            .first()
-        )
-        if context["attributes"].require_stable and operation:
+        operations = Operation.by_state(dataset, OperationState.WORKING)
+        if context["attributes"].require_stable and operations:
             raise APIAbort(
                 HTTPStatus.CONFLICT,
-                f"Dataset is working on {operation.name.name}",
+                f"Dataset is working on {','.join(o.name.name for o in operations)}",
+            )
+
+        component = context["attributes"].operation_name
+        try:
+            sync = Sync(logger=current_app.logger, component=component)
+            sync.update(dataset=dataset, state=OperationState.WORKING)
+            context["sync"] = sync
+        except Exception as e:
+            current_app.logger.warning(
+                "{} {} setting {} operational state: '{}'",
+                component,
+                dataset,
+                OperationState.WORKING.name,
+                e,
             )
```

Comment:
It's a `warning` and includes the exception text, so redundant additional verbosity seemed unnecessary. I'm not issuing a warning because everything worked ...